lite.js

```diff
@@ -3,194 +3,74 @@
 const rdf = require('rdf-ext')
 const SimpleArray = require('./lib/array')
 const SimpleContext = require('./lib/context')
+const SimpleCore = require('./lib/core')
+const SimpleHandler = require('./lib/define-property-handler')
 
 class SimpleRDF {
   constructor (context, iri, graph) {
-    this._iri = buildIri(iri)
-    this._objects = {}
+    this._core = new SimpleCore(this)
```

Comment:
I don't want to pollute the interface of the SimpleRDF class with all the properties and methods of Core and Handler. I agree that this is usually an anti-pattern, but we have a special use case. SimpleRDF is the base class for everything and we should try to keep everything small :-)